src/views/LeadAdd/ProcessingLeads/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { useCallback, useContext } from 'react';
```

Comment:
I think we should provide a name for candidates for lead creation.
and then name the component "ProcessingLead" to something like "LeadCandidatesPanel" or something?